src/ducks/index.jsx

```diff
@@ -1,9 +1,9 @@
 import { combineReducers } from "redux";
 
-import { testStateReducer } from "./testState/testStateReducer";
+import { authIntoSystemReducer } from "./authIntoSystem";
 
 const reducers = combineReducers ({
-    testState: testStateReducer
+    userInfo: authIntoSystemReducer
```

Comment:
I think its a good name `userInfo` section in the store. But why not to name duck the same?